### ISSUE TITLE
Should aria-valuetext function as an accessible name

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2655]
The YouTube volume slider does not have an accessible name, but uses aria-valuetext to convey its meaning. We may want to flag this for review instead of outright failing it. Needs to be tested in AT.

```
<div  role="slider" aria-valuemin="0" aria-valuemax="100" tabindex="0" aria-valuenow="100" aria-valuetext="100% volume">
```

**Comment by dylanb** [comment]
this is a mis-use of the attribute's semantics
